TAMUCTF2020/Reversing/AngrManagement.md

```diff
@@ -0,0 +1,23 @@
+# Angr Management
+
+## Description
+
+> nc challenges.tamuctf.com 4322
+
+The binary is provided.
+
+## Solution
+
+This challenge asks for a password, and checks it.
+
+![AngrManagement](../images/angrmanagement.png)
+
+By reversing the code, we see that the password is sent to 32 functions which check if it is correct or not. Each function can be easily reversed, consists mainly of simple mathematical conditions on the password.
+
+We use gdb to know at which function the verification fails:

```

Comment:
Expliquer qu'on fait trial and error, tu commences avec un string de la bonne longueur aaaaaaaa et tu vois ou ca fail et tu changes ta string a chaque iteration pour passer le check. C'est long et time consuming mais ca marche.